app/services/dialog_local_service.rb

```diff
@@ -12,6 +12,7 @@ class DialogLocalService
     InfraManager
     MiqGroup
     Service
+    ServiceAnsiblePlaybook
```

Comment:
I think we need to adjust the target type for this class, because when the API goes to look up a `ServiceAnsiblePlaybook`, it will not find it, because it needs to be simply looking up a `Service`.

This will also need a spec, but you can simply copy the existing spec for `Service` and change the class to `ServiceAnsiblePlaybook`

Reply:
@eclarizio please re-review